src/protocol/taiko_alethia/TaikoInbox.sol

```diff
@@ -61,10 +64,21 @@ contract TaikoInbox is IInbox {
         });
         require(metadata.anchorBlockHash != 0, "blockhash not found");
 
-        bytes[] memory attributes = new bytes[](3);
+        // Perform preemptively asserted queries
+        uint256 nQueries = queries.length;
+        uint256[] memory results = new uint256[](nQueries);
+        for (uint256 i = 0; i < nQueries; i++) {
+            require(queries[i].destination != address(publicationFeed), "Cannot query publication feed");
+            (bool success, bytes memory returndata) = queries[i].destination.call(queries[i].callData);
```

Comment:
For safety, should we use ```.staticcall``` instead of ```.call``` when executing L1 queries in ```TaikoInbox.publish``` here? We only need to read data.